docs/source/sysadmin_guide/configuring_sawtooth.rst

```diff
@@ -0,0 +1,37 @@
+********************
+Configuring Sawtooth
+********************
+
+Each Sawtooth component, such as the validator or the REST API, has an optional
+configuration file that controls the component's behavior. You can also specify
+configuration options on the command line when starting the component. For more
+information, see :doc:`/cli`.
+
+When a Sawtooth component it looks for a TOML configuration file (see
```

Comment:
Delete 'it'

Reply:
Actually, this was supposed to be "When a Sawtooth component starts, it looks for".  I've restored "starts,".